assets/css/mobile/pages/sunglasses.css

```diff
@@ -25,3 +25,42 @@ img.isInCartMark, img.loadingIcon {
     text-align: center;
     margin-bottom: .5em;
 }
+
+#mobile-shop-benefits-infographic {
+	position: relative;
+	width: 100%;
+}
+
+.benefits-container {
```

Comment:
`.mobile-shop-benefits-infographic__inner-container` would be better, because `.benefits-container` is a bit confusing - you don't know which benefits and you don't know that it's an inner container for `#mobile-shop-benefits-infographic`.
Also it would be better to have `mobile-shop-benefits-infographic` as a class name (not id), for consistency - so we know that everything is classes. With current implementation it's a bit confusing why you have id for the root container and classes for the inner container and its children.